smartsim/experiment.py

```diff
@@ -160,28 +174,50 @@ def __init__(
 
         self.exp_path = exp_path
 
-        self._launcher = launcher.lower()
-
-        if self._launcher == "auto":
-            self._launcher = detect_launcher()
-        if self._launcher == "cobalt":
-            raise SSUnsupportedError("Cobalt launcher is no longer supported.")
+        # TODO: Remove this! The contoller is becoming obsolete
+        self._control = Controller(launcher="local")
```

Comment:
We 100% can if no one is using the old implementations of methods that require it be present (e.g. if we can also remove `Experiment.start`, `Experiment.stop`, `Experiment.get_status`, etc.) 

I'm not currently, but I was afraid of making such a sweeping change as a side effect in this PR in case others on the core refactor were. If I get the go ahead from @juliaputko  and @amandarichardsonn I'd be more than willing to remove this attr in this PR!!

Reply:
I give a go ahead!